ReIndexOrchestrator/index.js

```diff
@@ -0,0 +1,44 @@
+const df = require('durable-functions');
+const moment = require('moment');
+
+module.exports = df.orchestrator(function* orchestratorFunctionGenerator(context) {
+  context.log('Starting ReIndexOrchestrator');
+  const input = context.df.getInput();
+
+  context.log({ input });
+
+  const indexNames = input.parameters.indexNames;
+  const apimApiName = input.parameters.apimApiName;
+  let indexDefinition = input.parameters.indexDefinition;
+
+  if (!indexDefinition) {
+    indexDefinition = yield context.df.callActivity('GetIndexDefinition', indexNames.active);
+  }
+  context.log(indexDefinition);
```

Comment:
Other log messages dealing with variables log json objects rather than just the variable. Should this also use that convention?